### PR TITLE
Add Alpakka banner

### PR DESCRIPTION
**Pull request by 2m**
that mentions a couple of frequently used sbt tasks.

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -3,7 +3,7 @@ import sbt._, Keys._
 object Dependencies {
 
   val ScalaVersions = Seq("2.11.11", "2.12.2")
-  val AkkaVersion = "2.4.17"
+  val AkkaVersion = "2.4.18"
```
Comment (hseeberger) [inline project/Dependencies.scala]: Why not 2.5.1?

Comment (2m) [inline project/Dependencies.scala]: We need to make sure that Alpakka works with 2.5.x and 2.4.x, therefore leaving the older version here. We could use 2.5.x here if we could run tests with jars from an older series.

Comment (johanandren) [inline project/Dependencies.scala]: In fact we do not know they do work with 2.5.1 as there are a few cases we are stricter with in graph stages now.

**Comment by 2m** [comment]
Validation failure was: https://github.com/akka/alpakka/issues/290

**Pull request merged**